runtime/doc/lsp.txt

```diff
@@ -1593,6 +1576,51 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (`integer`)
 
 
+==============================================================================
+Lua module: vim.lsp.completion                                *lsp-completion*
+
+accept_pum()                                 *vim.lsp.completion.accept_pum()*
+    Accept a completion item.
+
+    Return: ~
+        (`boolean`) `true` if the item was selected.
+
+attach({client_id}, {bufnr})                     *vim.lsp.completion.attach()*
+    Attaches the given client to the given buffer as a completion provider.
+
+    Parameters: ~
+      • {client_id}  (`integer`) Client ID
+      • {bufnr}      (`integer`) Buffer handle, or 0 for the current buffer
+
+detach({client_id}, {bufnr})                     *vim.lsp.completion.detach()*
+    Detaches a client from the given buffer to stop requesting LSP
+    completions.
+
+    Parameters: ~
+      • {client_id}  (`integer`)
+      • {bufnr}      (`integer`)
+
+omnifunc({findstart}, {base})                  *vim.lsp.completion.omnifunc()*
+    Implements 'omnifunc' compatible LSP completion.
+
+    Parameters: ~
+      • {findstart}  (`integer`) 0 or 1, decides behavior
+      • {base}       (`integer`) findstart=0, text to match against
+
+    Return: ~
+        (`integer|table`) Decided by {findstart}:
+        • findstart=0: column where the completion starts, or -2 or -3
+        • findstart=1: list of matches (actually just calls |complete()|)
+
+    See also: ~
+      • |complete-functions|
+      • |complete-items|
+      • |CompleteDone|
+
+trigger_completion()                 *vim.lsp.completion.trigger_completion()*
```

Comment:
is this analogous to `complete_check` or one of the other existing complete functions? Of course, if a new name is definitely needed then the door is open for that. But otherwise it helps discoverability to draw analogs to existing precedent.

In any case the function name probably doesn't need to mention "completion" since it exists on the `completion` module.
```suggestion
check()                 *vim.lsp.completion.check()*
```

Reply:
(or just "trigger", in any case)

Reply:
I'll change it to `trigger`.